File: php/examples/get_filing_products_offerings.php
<?php
require_once 'base_request.php';

// Example of GET /filing-products/offerings

$debug = true;
$jurisdiction = $_ENV['JURISDICTION'];
$company_id   = $_ENV['COMPANY_ID'];
$request_data = '';
$request_params = ['jurisdiction' => $jurisdiction, 'company_id' => $company_id];

send_request('GET', '/filing-products/offerings', $request_params, $request_data);

File: ruby/examples/get_filing_products_offerings.rb
require_relative '../request.rb'

# Example of GET /filing-products/offerings

COMPANY_ID	 = ENV['COMPANY_ID']
JURISDICTION = ENV['JURISDICTION']

params = {
  company_id: COMPANY_ID,
  jurisdiction: JURISDICTION
}

BaseRequestRoute.request(:get, '/filing-products/offerings', query_params: params)